src/main/java/dan200/computercraft/shared/pocket/apis/PocketAPI.java

```diff
@@ -78,11 +78,12 @@ public void shutdown()
                     {
                         if( !(m_computer.getEntity() instanceof EntityPlayer) )
                         {
-                            throw new LuaException( "Cannot find player" );
+                            return new Object[] { false, "Cannot find player" };
                         }
 
                         EntityPlayer player = (EntityPlayer) m_computer.getEntity();
                         InventoryPlayer inventory = player.inventory;
+                        if( !m_computer.inInventory() ) return new Object[] { false, "Not in an inventory" };
```

Comment:
That would be better, then I'm not entirely sure of the best course of action here. We can't really track when the pocket computer enters/exits a player's inventory - the only notification we get is the item's `update` method. 

Consequently, our only real strategy to determine whether the entity is valid is to scan the entire inventory for the pocket computer stack. We could add such a check to the `getEntity` method, returning `null` if it is not in our current entity's inventory. However, that does mean it is no longer thread-safe, which may be a problem in some situations.